pkg/helmfile/release_set.go

```diff
@@ -69,12 +69,6 @@ func NewReleaseSet(d ResourceRead) (*ReleaseSet, error) {
 	if env := d.Get(KeyEnvironment); env != nil {
 		f.Environment = env.(string)
 	}
-	// environment defaults to "" for helmfile_release_set but it's always nil for helmfile_release.
-	// This nil-check is required to handle the latter case. Otherwise it ends up with:
-	//   panic: interface conversion: interface {} is nil, not string
-	if path := d.Get(KeyPath); path != nil {
-		f.Path = path.(string)
-	}
```

Comment:
I'm going to merge this and make necessary changes(if any) later but one thing to ask here- What was the rationale behind removing this feature (specifying an existing path) in this pull request?

Reply:
@mumoshu When I started finding root cause of persisting tmp files - i found that there's two separate pieces of code creating tmp files for releases:
1) Creation of helmfile for standalone helm release
2) Copying existing helmfile before apply

This cases had completely different places where tmp files were created, so I refactored both cases to pass helmfile content same way and make same tmp file creation code for them.

When I sent this PR I tested that both terraform resources works fine with this PR, so my idea was unification of tmp file creation without breaking how terraform resources works